src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
Actually, the deviance is also computed by going over all observations. But I agree simplifying the code would be good. The issue is, as I mention in the second point of https://github.com/JuliaStats/StatsBase.jl/pull/146, I've not been able to compute the log-likelihood from the deviance as I expected and using the formula you mention. Any idea why?